.changeset/little-guests-hear.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
![image](https://github.com/user-attachments/assets/68aff721-e5ae-489d-8e83-6fb30c81c9e4)
The value does not match the Option, dog is capitalized but not in the selected value